src/web/core/src/main/java/org/geoserver/web/data/resource/AttributeTypeInfoEditor.html

```diff
@@ -11,6 +11,9 @@
     <wicket:fragment wicket:id="area">
         <textarea wicket:id="area" class="cql"></textarea>
     </wicket:fragment>
+    <wicket:fragment wicket:id="description">
+        <textarea wicket:id="description" class="cql"></textarea>
+    </wicket:fragment>
```

Comment:
Making the table wider and wider is not working out so well. Description in particular invites an entire sentence to be written (if not a paragraph). 

I do not have a good/cheap suggestion on how to address this feedback - any ideas?

Reply:
Is this really ``class="cal"`` should it not be text of some form?

Reply:
We don't have resources to make a better editor, but something radically different will be needed in case international descriptions are actually used (thinking out loud, a pop-up editor of some sort, or remove the table based editor and go for a list-based one instead).

We can go as far as using InternationalString in the API, changing the editor and making actual usage of international strings can be implemented later by parties having both interest and funds. This will involve also figuring out the current language in the DescribeFeatureType response and act accordingly, that is also not currently internationalized.

Reply:
Thanks, changing the api is a good step (and good communication of intent).